pkg/cosign/types.go

```diff
@@ -0,0 +1,11 @@
+package cosign
+
+const (
+	SignatureAnnotationSignedName = "acorn.io/signedName" // If an image was signed by `acorn image sign foo/bar:v1`, this annotation should be set to `foo/bar:v1` (the payload usually only includes the image digest)
```

Comment:
Looks like the k8s naming convention for annotation keys calls for `lower-kebab-case` (see https://github.com/kubernetes/api/blob/master/core/v1/annotation_key_constants.go for precedent). Could we change this to align before merging?

```suggestion
	SignatureAnnotationSignedName = "acorn.io/signed-name" // If an image was signed by `acorn image sign foo/bar:v1`, this annotation should be set to `foo/bar:v1` (the payload usually only includes the image digest)
```

Reply:
It's not a k8s annotation, but I don't mind the case here :man_shrugging: